### ISSUE TITLE
[Discussion] - How to make the voting system secure?

### ISSUE THREAD

**Issue by ppconde** [esnaveiro/voting-booth#2]
Possible approach using blockchain:
https://github.com/mehtaAnsh/BlockChainVoting

EDIT:
Interesting reads:
https://web3auth.io/docs/guides/firebase
https://moralis.io/web3-firebase-authentication-create-a-web3-sign-in-with-moralis/

Open to ideas. Leave your suggestions below.

**Comment by Web-Project-Administrator** [comment]
I liked very much the system presented on the first link. The fact that the admin can see how many people voted (and who because of the emails) helps the flow of the voting.
Regarding the system security and authentication, there are a lot more things we can do to improve the overall funcionality of the app before going into the security and authentication. My sugestion would be to pick up this issue later when the structure is more developed. 
Tell me your thoughts.

**Comment by ppconde** [comment]
I agree. Let's keep it simple for now, but it might interesting for later
